yr/scripts/tutorialPage.js

```diff
@@ -245,8 +245,8 @@ var setupTutorial = function() {
   });
 
   // Add show/hide hint functionality
-  $(".hintContainer").off("click").on("click", function(e) {
-    $(".hint").toggleClass("hideHint");
+  $(".hintContainer>button").off("click").on("click", function(e) {
```

Comment:
Now that I've had a little more time to think about this, I think instead of using `button` here we should use `*:first-child`. This way, we can decide to move to another element for showing the hint (e.g., a div) without having to touch the code.